### DISCUSSION TITLE
Daily schedule?

### DISCUSSION THREAD

**Discussion by Reflexeye** [gopro/labs#860] [Feature Requests]
Hi - wondering if poss to schedule long term timelapse to take images between sunrise and sunset only...or between sunset and sunrise (star lapses)?

**Comment by dnewman-gpsw** [comment]
Yes, part of Labs since the HERO8 Labs introduction.  All actions can be time based, including time deltas from sunset of sunrise.  Some documentation here: https://gopro.github.io/labs/control/actions/

I'm personally running a very longform timelapse, on 3 photos a day, one photo 30 minute before sunset, one 30 minutes after sunrise and one at noon, everyday, automatically.

dPmP!s-1800N!S!r1800N!S!12:00N!S!R  <- this will run ~50days on internal battery alone (HERO12)

You can start and end a timelapse the same way.

mNL!sS!rE   <- mode NightLapse, start at sunset, end at sunrise.

**Comment by Reflexeye** [comment]
Thanks, David.
I'll try and modify your script to take photos every 10 minutes between
sunrise and sunset.
I used the app to generate a QR code, but alas it doesn't seem to have
worked...although there is a section for starting a sequence, there's no
section for ending a sequence of photos around solar time.
atb
Jem

On Sun, 14 Apr 2024 at 19:37, David Newman ***@***.***> wrote:

> Yes, part of Labs since the HERO8 Labs introduction. All actions can be
> time based, including time deltas from sunset of sunrise. Some
> documentation here: https://gopro.github.io/labs/control/actions/
>
> I'm personally running a very longform timelapse, on 3 photos a day, one
> photo 30 minute before sunset, one 30 minutes after sunrise and one at
> noon, everyday, automatically.
>
> dPmP!s-1800N!S!r1800N!S!12:00N!S!R <- this will run ~50days on internal
> battery alone (HERO12)
>
> You can start and end a timelapse the same way.
>
> mNL!sS!rE <- mode NightLapse, start at sunset, end at sunrise.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/gopro/labs/discussions/860#discussioncomment-9110857>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BH2HEBUIBKPDTB4GFANQ4DTY5LEG5AVCNFSM6AAAAABGGHNP2OVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TCMJQHA2TO>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Jem.

**Reply by dnewman-gpsw** [reply]
You have to do some of this in the script language manually.
